### ISSUE TITLE
Editing PNG images reduces image quality

### ISSUE THREAD

**Issue by Asiaboy89** [SimpleMobileTools/Simple-Gallery#3042]
I'm working often with screenshots (PNGs) and small texts within the screenshots and after adding some comments/text by myself via the editor, the screenshot is noticably worse than before. When I do this 3 or 4 times with the same screenshot (means: adding text, saving photo, adding text, saving, etc.), the content of the screenshot is barely readable.

This issue was not always present, only after updating the app to a newer version 1,5 years ago or so.

Example:
Original
![original](https://github.com/SimpleMobileTools/Simple-Gallery/assets/67432501/b66c7dda-52b1-4543-a270-8da2999752d7)

![original_1](https://github.com/SimpleMobileTools/Simple-Gallery/assets/67432501/1e654498-e112-46ea-81f6-cc0af564d14a)
![original_1_1](https://github.com/SimpleMobileTools/Simple-Gallery/assets/67432501/7fc9ab50-c75e-47a3-ae32-c706bfc20100)
![original_1_1_1](https://github.com/SimpleMobileTools/Simple-Gallery/assets/67432501/9d878318-629c-479f-b655-696ad4de9a72)

**Comment by Aga-C** [comment]
It's a duplicate of https://github.com/SimpleMobileTools/Simple-Gallery/issues/2349.
